erpnext/controllers/stock_controller.py

```diff
@@ -1661,7 +1661,7 @@ def future_sle_exists(args, sl_entries=None, allow_force_reposting=True):
 		from `tabStock Ledger Entry` force index (item_warehouse)
 		where
 			({})
-			and timestamp(posting_date, posting_time)
+			and posting_datetime
 				>= timestamp(%(posting_date)s, %(posting_time)s)
```

Comment:
Need to remove this timestamp function as well

Reply:
I thin Mariadb clever enough so that it doesn't calculate that for each row. But I will.